tests/containers/validate_btrfs.pm

```diff
@@ -45,13 +45,18 @@ sub _sanity_test_btrfs {
     assert_script_run("echo -e 'FROM $img\\nENV WORLD_VAR Arda' > $dockerfile_path/Dockerfile");
     my $btrfs_head = '/tmp/subvolumes_saved';
 
+    my $storage = $rt->get_storage_driver();
     if (version->parse(get_docker_version()) >= version->parse('23.0.5')) {
-        $rt->info(property => 'Driver', value => 'overlay2');
+        if ($storage ne 'overlay2') {
```

Comment:
Since this is used for `podman` and `docker`, don't we have to also allow `overlay`, as being used by `podman` as alias for `overlay2`? :thinking:

Reply:
That's true, in this case it works as expected.